samples/snippets/README.md

```diff
@@ -1,87 +0,0 @@
-# Google Cloud Tasks Samples
```

Comment:
Could you replace this with something like https://github.com/googleapis/python-kms/tree/main/samples/snippets

Reply:
Thank you for the reference @dandhlee